### PR TITLE
ROX-18211: Delete Network Graph 1.0 styles and leftover fonts

### PR DESCRIPTION
**Pull request by pedrottimark**
## Description

1. Edit package.json: delete font package dependencies overlooked
    * typeface-open-sans in #5977
    * typeface-open-sans-condensed in #6113

2. Edit src/app.tw.css: find the following, and then notice some other obsolete class names
    * cytoscape 1
    * network 8
    * simulator 4

3. Edit src/css/style.css: delete stackrox font after
    * references as font-family deleted in #6699
    * last reference to 1 of its 3 custom characters was deleted in #6825

    ```js
    name: `${isActive ? '\ue901 ' : ''}${namespace}`,
    ```

4. Delete src/fonts folder

### Residue

1. What is overpass font from PatternFly?
    * `wc build/static/media/RedHat*.*` 665085
    * `ls -al build/static/media/RedHat*.* | wc` 22 files
    * `wc build/static/media/overpass-*.*` 1727296
    * `ls -al build/static/media/overpass-*.* | wc` 40 files

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

## Testing Performed

TL;DR although small savings, it is in main.css bundle

1. `yarn build` in ui and then compute branch - master for the following
    * `wc build/static/css/*.css`
        main.css: -2566 = 4972281 - 4974847
        total: -2566 = 5821187 - 5823753
    * `ls -al build/static/css/*.css | wc`
        0 = 28 - 28 files
    * `wc build/static/media/*.*`
        -14147 = 4462350 - 4476497
    * `ls -al build/static/media | wc`
        -3 = 141 - 144 files
3. `yarn start` in ui

### Manual testing

1. Visit /main/dashboard

2. Click **Network Graph** in left navigation to visit /main/network-graph
    network-graph_all

### CODE REVIEW

**Review by dvail — APPROVED** [review]
Residue research:

"Overpass" is an original Red Hat font that was included as part of PF 4 as a fallback for users that didn't want to use "Red Hat font". It looks like this will [naturally drop away when we upgrade to PF 5](https://github.com/patternfly/patternfly/issues/4219).
